e2e/scenario_test.go

```diff
@@ -892,6 +892,7 @@ func Test_ubuntu2204gpuNoDriver(t *testing.T) {
 }
 
 func Test_ubuntu2204privatekubepkg(t *testing.T) {
+	t.Skip("this VHD can't be updated and self-contained logic doesn't work for it")
```

Comment:
we probably don't want to blindly disable this, would there be a way we could keep it around for non self-contained?

Reply:
Can be done with adding `if config.EnableNodeBootstrapperTest`

Reply:
Have removed this skip and added a way for tests to specify which bootstrapping method they want.